packages/react/src/components/_CheckboxRadioTemplate/CheckboxRadioTemplate.tsx

```diff
@@ -30,6 +30,11 @@ export interface CheckboxRadioTemplateProps {
   size: 'small' | 'xsmall';
   type: 'checkbox' | 'radio';
   value?: string;
+  /**
+   * Should only be used when you want to render
+   * your own label as a parent to this component.
+   */
+  dontRenderLabelTag?: boolean;
```

Comment:
That makes more sense!

Reply:
I agree with @TomasEng, but what about refactoring the code to avoid rendering the label if the label is not provided? It feels wired to pass in a label and at the same time add "shouldRenderLabel=false".

What do you think? :)

Reply:
Hm. Well in this case we actually _do_ pass in a `label`-prop, we simply don't want that prop to be wrapped in a `label`-tag in the DOM. I see that the `input` also has an `aria-labelledby` attribute, which points to a `span` with the `label`-prop text. Maybe we need to look into how this should be labelled.

Reply:
We changed the implementation, ref https://github.com/digdir/designsystem/pull/562#issuecomment-1596795207